app/src/main/java/com/example/coronadiagnosticapp/ui/views/FeelingQuestionView.kt

```diff
@@ -40,10 +40,20 @@ class FeelingQuestionView : LinearLayout {
     private fun addFeelings() = GeneralFeeling.values()
         .forEach { addFeeling(it) }
 
+    private fun getLocalizedFeeling(feeling: String): CharSequence? {
+        var res = ""
+        when (feeling) {
+            "Same" -> res = context.getString(R.string.same)
+            "Better" -> res = context.getString(R.string.better)
+            "Worse" -> res = context.getString(R.string.worse)
+        }
+        return res
+    }
+
     private fun addFeeling(feeling: GeneralFeeling) {
         val btn = EkRadioBtn(context).apply {
             id = feeling.ordinal
-            text = feeling.name.toLowerCase().capitalize()
+            text = getLocalizedFeeling(feeling.name.toLowerCase().capitalize())
```

Comment:
That's a good point. we can add an enum field instead which represents the ResId int
and then just getString with it like so:
res = context.getString(feeling.resId)